lib/git_reflow/git_server/pull_request.rb

```diff
@@ -3,6 +3,16 @@ module GitServer
     class PullRequest
       attr_accessor :description, :html_url, :feature_branch_name, :base_branch_name, :build_status, :source_object, :number
 
+      LGTM_REGEX = /(?i-mx:lgtm|looks good to me|[:\\+1:]|:thumbsup:|:shipit:)/
```

Comment:
I agree, catching myself from our previous discussion.  No need to set a default on minimum as that is not the current behavior.  Ignore my recommendation here :wink: